### ISSUE TITLE
Is settled/in progress functionality?

### ISSUE THREAD

**Issue by pablohirafuji** [Gizra/elm-debouncer#14]
Great package and well thought API, it's been very useful!

I wonder if there is a way to know if the debouncer is settled or not.

I want to show a loading when it is not settled (in progress), without having to manage another state, like `isLoading : Bool` and maintain them in sync.

Sorry if it's already possible, but I couldn't figure it out.

**Comment by rgrempel** [comment]
I don't think it is possible at the moment, with the current API. However, it makes a lot of sense, and wouldn't be difficult to add, so I'll do that.
